_includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

```diff
@@ -0,0 +1,2544 @@
+| HoppingList                 | vendor/spryker/shopping-list/               |
+| HoppingListNote             | vendor/spryker/shopping-list-note/          |
+| HoppingListProductOptionConnector | vendor/spryker/shopping-list-product-option-connector/ |
+| Tock                        | vendor/spryker/stock/                       |
+| TockDataImport              | vendor/spryker/stock-data-import/           |
+| TorageDatabase              | vendor/spryker/storage-database/            |
+| TaxProductStorage           | vendor/spryker/tax-product-storage/         |
+| TaxStorage                  | vendor/spryker/tax-storage/                 |
+| Twig                        | vendor/spryker/twig/                        |
+| Wishlist                    | vendor/spryker/wishlist/                    |
+| WishlistsRestApi            | vendor/spryker/wishlists-rest-api/          |
+|-----------------------------|---------------------------------------------|
+
+{% endinfo_block %}
+
+
+
+{% info_block warningBox "Verification Spryker-Shop modules" %}
+
+Make sure that the new modules have been installed or updated in the following directories:
+
+| MODULE                      | EXPECTED DIRECTORY                          |
+|-----------------------------|---------------------------------------------|
+| CatalogPage                 | vendor/spryker-shop/catalog-page/           |
+| CheckoutPage                | vendor/spryker-shop/checkout-page/          |
+| CmsPage                     | vendor/spryker-shop/cms-page/               |
+| CompanyPage                 | vendor/spryker-shop/company-page/           |
+| ConfigurableBundleWidget    | vendor/spryker-shop/configurable-bundle-widget/ |
+| CustomerPage                | vendor/spryker-shop/customer-page/          |
+| CustomerReorderWidget       | vendor/spryker-shop/customer-reorder-widget/ |
+| LanguageSwitcherWidget      | vendor/spryker-shop/language-switcher-widget/ |
+| MoneyWidget                 | vendor/spryker-shop/money-widget/           |
+| ProductMeasurementUnitWidget | vendor/spryker-shop/product-measurement-unit-widget/ |
+| ProductNewPage              | vendor/spryker-shop/product-new-page/       |
+| ProductPackagingUnitWidget  | vendor/spryker-shop/product-packaging-unit-widget/ |
+| ProductSearchWidget         | vendor/spryker-shop/product-search-widget/  |
+| QuickOrderPage              | vendor/spryker-shop/quick-order-page/       |
+| ShopApplication             | vendor/spryker-shop/shop-application/       |
+| ShopUi                      | vendor/spryker-shop/shop-ui/                |
+| ShoppingListPage            | vendor/spryker-shop/shopping-list-page/     |
+| StoreWidget                 | vendor/spryker-shop/store-widget/           |
+
+{% endinfo_block %}
+
+### Set up database schema and transfer objects
+
+Apply database changes and generate entity and transfer changes:
+    
+```bash 
+
+console transfer:generate
+console propel:install
+console transfer:generate
```

Comment:
if we need this to generate only entity transfer I believe we had a separate command

Reply:
We do this in order to generate propel entity transfers, so it is fine.